client/src/components/common/Button/Button.jsx

```
@@ -12,22 +44,137 @@
+					}
+				} else if (showNotifications) {
+					setNotification({
+						open: true,
+						message: 'No URL specified for link action',
+						severity: 'warning',
+					});
+				}
+				break;
+
+			case 'addToCart':
+				if (cartContext && cartContext.addToCart && productId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)